crates/pop-common/src/lib.rs

```diff
@@ -73,6 +73,16 @@ pub fn find_free_port() -> u16 {
 		.port()
 }
 
+/// Provides functionality for making calls to parachains or smart contracts.
+pub mod call {
+	// Note: cargo contract logic is used for parsing events after calling a chain. This could be
```

Comment:
This can be a potential refactor when we find the time: `refactor: event handling`